mbtiles/src/summary.rs

```diff
@@ -303,7 +303,7 @@ mod tests {
               - -123.75000000000001
               - -40.97989806962013
               - 179.99999999999997
-              - 61.60639637138628
+              - 61.60639637138627
```

Comment:
Not quite sure what this test run change is about, so I want to look into it, even if the precision level here is negligible.

Reply:
In CI at least, this is the following..

```suggestion
              - 61.60639637138628
```

Reply:
Just updated the other line below this one as well. Will verify in CI build